docs/samples/pytorch/pytorch.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: "serving.kubeflow.org/v1alpha1"
+kind: "KFService"
+metadata:
+  name: "pytorch-cifar10"
+spec:
+  default:
+    pytorch:
+      modelUri: "gs://tomcli-test/"
+      modelClassName: "Net"
+      modelClassFile: "cifar10.py"
```

Comment:
not sure how what's the best practice for pytorch to manage model class file, does user has to manually upload this file to the storage? What if user updated the model but not the file?

Reply:
Yes, the model class file, and the model file come as part of a single package

Reply:
Maybe could omit modelClassFile and instead just get users to provide a fully qualified class name? In this example that would be "cifar10.Net"?

Reply:
Discussed with ellis - simplified to prescribe one model file, which we discover, and taken an optional model class name

Reply:
Done